circleguard/circleguard.py

```diff
@@ -98,15 +101,28 @@ def run(self, check):
             comparer = Comparer(d.steal_thresh, compare1, replays2=compare2)
             yield from comparer.compare()
 
+        # relax check
         if Detect.RELAX in d or Detect.CORRECTION in d:
+            if Detect.RELAX in d:
+                if self.library is None:
```

Comment:
`if not self.library`

don't directly compare to `None` unless you need to distinguish between `None` and falsey, which you don't here.